services/armremotecontrol/builtin/builtin_test.go

```diff
@@ -65,7 +66,10 @@ func TestArmRemoteControl(t *testing.T) {
 		case input.Subtype:
 			return fakeController, nil
 		case arm.Subtype:
-			return fakearm.NewArmIK(ctx, config.Component{Name: "arm"}, logger)
+			return fakearm.NewArm(ctx, config.Component{
+				Name:                arm.Subtype.String(),
+				ConvertedAttributes: &fakearm.AttrConfig{ArmModel: xarm.ModelName(6)},
```

Comment:
I assume this change is so that this test can use the kinematic model of the xArm6?

Reply:
Yes, before it was using another fake kinematic model, which no longer exists